recipes/pynnotator/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pynnotator" %}
+{% set version = "1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4e777813fec487e4352ad4fd6bd55ae5286a120fab848cdba7c52c126645aab5"
+
+build:
+  number: 0
+  entry_points:
+    - pynnotator=pynnotator.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python 
+
+requirements:
+  host:
+    - Cython
```

Comment:
You want "cython", which means you need a C compiler and this can't be a noarch package.

Reply:
Yes, I want cython for pysam. I got the idea from `bioconda-utils lint --git-range master` that I should add Cython instead of cython. That's fixed.